quiffen/core/transaction.py

```diff
@@ -21,6 +21,16 @@
 logger = logging.getLogger(__name__)
 
 
+class TransactionConfig:
+    """
+    transaction configuration settings
+    """
+
+    # maximum percentage overshoot of a split transaction above
+    # 100%
+    max_split_overshoot: float = 0.01
+
```

Comment:
I'm not sure I'm a fan of storing config on a class like this. Also, if this is intended to be user-editable, then perhaps there's a better way of approaching this than a global config object.